tests/conftest.py

```diff
@@ -206,9 +211,28 @@ def init_asicdb_validator(self):
     def runcmd(self, cmd):
         return self.ctn.exec_run(cmd)
 
+    def get_file(self, docker_path, local_filename):
+        output = self.ctn.exec_run("cat %s" % docker_path)
+        with open(local_filename, "w") as fp:
+            fp.write(output)
+
 @pytest.yield_fixture(scope="module")
 def dvs(request):
     name = request.config.getoption("--dvsname")
+    get_logs = request.config.getoption("--get_logs")
+    orch_log_level = request.config.getoption("--orchagent_loglevel")
+
     dvs = DockerVirtualSwitch(name)
+    dvs.ctn.exec_run("swssloglevel -l %s -c orchagent" % orch_log_level)
+
     yield dvs
+
+    if get_logs:
```

Comment:
this is getting all logs. Usually, when you done with an appdb input, you just want to get new logs. I think we need to provide api to do that.

Reply:
What do you mean by "new logs"? Do you mean we want to see only logs which were created during a test?

Reply:
I mean new log entries generated after you run certain commands.